unattended_scripts/install_functions/opendistro/filebeat.sh

```diff
@@ -45,7 +45,7 @@ configureFilebeat() {
         echo "  - ${nip}"  >> /etc/filebeat/filebeat.yml
     else
         echo "output.elasticsearch.hosts:" >> /etc/filebeat/filebeat.yml
-        sh=$(awk -v RS='' '/discovery.seed_hosts:/' ./config.yml)
+        sh=$(awk -v RS='' '/discovery.seed_hosts:/' ${base_path}/config.yml)
```

Comment:
Change this in kibana.yml, common.yml and elasticsearch.yml too. When using files from S3, when is this file downloaded?

Reply:
Done